views/shared/_info-panel.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="row">
+  <div class="col button">
+    <button type="button" class="btn btn-primary" disabled>
+      Level:
+      <span class="badge badge-light">
+                   <%= difficulty %>
+                </span>
+    </button>
+    <button type="button" class="btn btn-primary" disabled>
+      Attempts Left:
```

Comment:
All strings in views that may be translated should be in I18n

Reply:
+